examples/step-69/step-69.cc

```diff
@@ -2822,14 +2822,12 @@ namespace Step69
 
     schlieren_postprocessor.compute_schlieren(output_vector);
 
-    auto data_out = std::make_shared<DataOut<dim>>();
-
+    std::unique_ptr<DataOut<dim>> data_out = std::make_shared<DataOut<dim>>();
```

Comment:
Am I missing something or why aren't you using 
```suggestion
    std::unique_ptr<DataOut<dim>> data_out = std::make_unique<DataOut<dim>>();
```

Reply:
You didn't miss anything. It took too long to compile on the laptop and so I decided to let the CI infrastructure do it.

Reply:
In any case, fixed now.